worker/peergrouper/desired_test.go

```diff
@@ -409,6 +524,7 @@ var stateFlags = map[rune]replicaset.MemberState{
 // 	- 'H' if the instance is not healthy.
 //	- 'p' if the instance is in PrimaryState
 //	- 's' if the instance is in SecondaryState
+// TODO (agprado 3/5/18): check there's never more than one primary
```

Comment:
It would be good to do this, just to help ensure our test cases are actually valid.